src/Raven.Server/Smuggler/Documents/Handlers/SmugglerHandler.cs

```diff
@@ -54,7 +54,7 @@ public Task PostValidateOptions()
                 var exporter = new SmugglerExporter(Database, options);
                 var token = CreateOperationToken();
 
-                var fileName = string.IsNullOrEmpty(exporter.Options.FileName) ?
+                var fileName = string.IsNullOrEmpty(exporter.Options.FileName) || exporter.Options.FileName.IndexOfAny(Path.GetInvalidFileNameChars()) >= 0 ?
```

Comment:
If the filename is invalid, we don't care. We are sending it to the browser, to handle that.
More to the point, this code is just asking to get a bug report about "ravendb is ignoring my custom name"